snowflake/1.1/modules/ROOT/pages/snowflake-connector-studio.adoc

```diff
@@ -0,0 +1,142 @@
+= Use Anypoint Studio to Configure Snowflake Connector 1.1 - Mule 4
+
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:snowflake-connector-reference.adoc[Snowflake Connector Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add Snowflake Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In *Mule Palette*, click *(X) Search in Exchange*.
```

Comment:
We have been saying "In the *Mule Palette* view," and I see you used those words in the example. I wouldn't change this, since it's not a big deal. Just for going forward.